lib/physicians/validation.ts

```diff
@@ -0,0 +1,31 @@
+import { PatientCondition, PermitType } from '@prisma/client';
+import { date, mixed, object, string } from 'yup';
+
+/**
+ * Validation schema for physician assessment form
+ */
+export const physicianAssessmentSchema = object({
+  disability: string().required('Please enter a disabling condition'),
+  disabilityCertificationDate: date().required('Please enter a valid certification date'),
```

Comment:
I added in a max value for this, but it will require testing on a deployed instance to verify it works. I've seen some discussions about yup storing the date as the build date and not recalculating it every time ([example](https://stackoverflow.com/questions/63381887/yup-compares-min-max-date-with-build-date-instead-of-current-date)), but other discussions indicate that it should work ([example](https://stackoverflow.com/questions/54917648/yup-formik-min-date-does-not-include-current-date)). Should be fine as is for now and once we deploy the whole branch we can verify this. I'll add a TODO for this

Reply:
Sounds good!